topics/galaxy-interface/tutorials/galaxy-intro-jupyter/tutorial.md

```diff
@@ -48,93 +58,259 @@ You can find the complete manual for Jupyter commands [on Read the Docs](http://
 
 # Use Jupyter notebook in Galaxy
 
-## Open a Notebook
-The Jupyter notebook can be started from different points. You can either open a Jupyter notebook from a dataset in your history or from the *Visualize* tab in the upper menu.
-
+## Import data
+To manipulate data first we upload the  [python-novice-gapminder-data.zip](https://swcarpentry.github.io/python-novice-gapminder/files/python-novice-gapminder-data.zip) folder into your Galaxy history. 
 
-> <hands-on-title>Launching a Jupyter notebook from a dataset or a saved Jupyter notebook</hands-on-title>
-> If you only need one dataset from your history to perform you analysis or want to open a Jupyter notebook that you previously saved in your history, you can launch a Jupyter from a single dataset.
-> 1. Expand the dataset in you history by clicking on its name.
-> 2. Click on the visualization icon {% icon galaxy-barchart %} of the dataset `[...].counts`.
-> 3. Select the Jupyter visualization in the list. ![Visualisation list](../../images/visu_list.png)
+> <hands-on-title>Data upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import [python-novice-gapminder-data.zip](https://swcarpentry.github.io/python-novice-gapminder/files/python-novice-gapminder-data.zip) folder. The folder contains the following tabular files:
+>    - `gapminder_all.csv` 
+>    - `gapminder_gdp_africa.csv` 
+>    - `gapminder_gdp_americas.csv`
+>    - `gapminder_gdp_asia.csv`
+>    - `gapminder_gdp_europe.csv`
+>    - `gapminder_gdp_oceania.csv`
+>
+>    ```text
+>    https://swcarpentry.github.io/python-novice-gapminder/files/python-novice-gapminder-data.zip
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
 >
+>
+> 3. Unzip data using the {% tool [Unzip](toolshed.g2.bx.psu.edu/repos/imgteam/unzip/unzip/6.0+galaxy0) %}
```

Comment:
We could safe this step if we just re-upload the data to Zenodo

Reply:
Thanks for the suggestion. I did add data to Zenodo and updated the tutorial. If @bgruening, @Delphine-L or @shiltemann have some extra time I would appreciate a review. Thanks a lot!